var/spack/repos/builtin/packages/sz/package.py

```diff
@@ -97,87 +97,66 @@ def setup_run_environment(self, env):
         if "+hdf5" in self.spec:
             env.prepend_path("HDF5_PLUGIN_PATH", self.prefix.lib64)
 
-    def _test_2d_float(self):
+    def test_2d_float(self):
         """This test performs simple 2D compression/decompression (float)"""
```

Comment:
The docstring is the brief description of the test so should be more concise (e.g., run simple 2D compression/decompression).